airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQuerySqlGenerator.java

```diff
@@ -179,7 +188,7 @@ public String createTable(final StreamConfig stream, final String suffix) {
             """
             CREATE SCHEMA IF NOT EXISTS ${final_namespace};
 
-            CREATE TABLE ${final_table_id} (
+            CREATE OR REPLACE TABLE ${final_table_id} (
```

Comment:
@edgao calling out that this wasn't exactly part of my mission with this but if a sync fails or is cancelled and the tmp table is never cleaned up, without the `REPLACE` statement the sync will fail. Let me know if this is too sneaky/dangerous